docs/restrict-cross-namespace-associations.asciidoc

```diff
@@ -6,22 +6,22 @@ endif::[]
 [id="{p}-restrict-cross-namespace-associations"]
 === Restrict cross namespace resource associations
 
-By default, associations can be created between any resources as long as they are deployed in some namespaces managed by the same ECK instance.
+By default, associations can be created between any resources as long as they are deployed in namespaces managed by the same ECK instance.
 
-This is true, for example, when using the `elasticsearchRef` field in order to automatically configure and establish a trust relationship between Kibana and Elasticsearch. If a user can create a Kibana instance in a namespace managed by ECK then this Kibana instance can be associated with any Elasticsearch cluster deployed in another namespace managed by the same ECK instance.
+This is true, for example, when using the `elasticsearchRef` field to automatically configure and establish a trust relationship between Kibana and Elasticsearch. If a user can create a Kibana instance in a namespace managed by ECK, then this Kibana instance can be associated with any Elasticsearch cluster deployed in another namespace managed by the same ECK instance. Even if the user only has permissions to edit Kibana and not the Elasticsearch resource.
```

Comment:
I _think_ the additional sentence starting with "Even if" is not corrent. IIUC "Even if" is a dependent clause and you need to have another clause in that sentence it cannot stand on its own. So either add another main clause or join it to the preceding sentence.